function/fetchFile.js

```diff
@@ -0,0 +1,79 @@
+/* Form of a Google Cloud Function URL (https://cloud.google.com/functions/docs/calling/http)
+   "https://YOUR_REGION-YOUR_PROJECT_ID.cloudfunctions.net/FUNCTION_NAME"
+*/
+
+//HTML that saves the breed and score from the returned JSON body
+<div id="prediction-results" style="display: none">
+<span class="label">Breed: </span><span id="dog-breed" class="score"></span> //Can access with getElementById
+<span class="label">Confidence: </span><span id="prediction-score" class="score"></span> //Can access with getElementById
+</div>
+
```

Comment:
These need to be in the index.html doc, as this JS file should only be JavaScript code.

Reply:
Okay! I'm about to have an interview from 1:00 - 1:30, but I can move these items afterward! Also, do we still have a team meeting later today?

Reply:
Done!